### ISSUE TITLE
World's bank data in examples has non-standard country codes

### ISSUE THREAD

**Issue by Pyriphlegethon** [airbnb/caravel#270]
The country codes KSV, ADO, ZAR, WBG, ROM, IMY, TMP and CHI don't work correctly with the world map visualization as they are always rendered in the top left corner of the map. Fixing this should only require adding them in [countries.py](https://github.com/airbnb/caravel/blob/master/caravel/data/countries.py).

**Comment by mistercrunch** [comment]
What standard are those country codes part of?

**Comment by Pyriphlegethon** [comment]
I don't know, they're the codes in the country_code column of the World Health Bank database and at least one of them is quite a big country (ZAR is the Democratic Republic of the Congo, the second biggest country in Africa).

**Comment by mistercrunch** [comment]
Weird, here's Congo with different codes. 
https://github.com/airbnb/caravel/blob/master/caravel/data/countries.py#L1439

This matches this other source of ISO country codes:
www.nationsonline.org/oneworld/country_code_list.htm

I'm not sure what's off with the World's Bank data...

**Comment by mukkoo** [comment]
@Pyriphlegethon  I've searched these codes in World Bank Health http://data.worldbank.org/topic/health.
I've found COD for Congo (as is already implemented).
I've found KSV for Kosovo that is KOS in Caravel.
I've not found  in World Bank Health ADO, ZAR, WBG, ROM, IMY, TMP.
I've found CHI for "Channel Islands" that is for "Chile" in Caravel in "cioc" key.

Searching for a solution I've found that World Bank seems to use this standard https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3 but in their datas returns non standard keys.
